src/main/java/bot/tasks/Event.java

```diff
@@ -0,0 +1,92 @@
+package bot.tasks;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.regex.Pattern;
+
+/**
+ * Represents a task that starts at a specific time and ends at a specific time.
+ */
+public class Event extends Task {
+    protected static final String YMD_DATE_FORMAT = "yyyy/MM/dd";
```

Comment:
Would it be better if these constant is in Task so both Event and Deadline can access it without being copied?

Reply:
That would be better! Will change this